openedx/core/djangoapps/coursegraph/management/commands/tests/utils.py

```diff
@@ -63,38 +83,18 @@ def create(self, element):
         if isinstance(element, Node):
             self.temp.add(element)
 
-    def commit(self):
-        """
-        Takes elements in the transaction's temporary storage and adds them
-        to the mock graph's storage. Throws an error if the graph's
-        transaction_errors param is set to True.
-        """
-        if self.graph.transaction_errors:
-            raise Exception("fake exception while trying to commit")
-        for element in self.temp:
-            self.graph.nodes.add(element)
-        self.temp.clear()
-        self.graph.number_commits += 1
-
-    def rollback(self):
-        """
-        Clears the transactions temporary storage
-        """
-        self.temp.clear()
-        self.graph.number_rollbacks += 1
-
 
-class MockNodeSelector:
+class MockNodeMatcher:
```

Comment:
All API names based on the term "selector" are now based on the term  "matcher". Otherwise, these classes and functions work the same.